.github/workflows/ci_test-slow.yml

```diff
@@ -0,0 +1,83 @@
+name: Test
+
+# see: https://help.github.com/en/actions/reference/events-that-trigger-workflows
+on:  # Trigger the workflow on push or pull request, but only for the master branch
+  push:
+    branches: [master, "release/*"]
+  pull_request:
+    branches: [master, "release/*"]
+
+jobs:
+  slow:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macOS-latest]
```

Comment:
just noticed that here we skip run on Windows and all minimal requirements?

Reply:
Yes, I wanted to keep this light at first as we don't run many tests at the moment. These can be added later